docs/source/technical_tutorials/user_data/stac_metadata.md

```diff
@@ -0,0 +1,60 @@
+# MAAP STAC Metadata Recommendations
+MAAP uses the STAC (SpatioTemporal Asset Catalog) specification, which provides a common structure for describing and cataloging geospatial data [(STAC)](https://stacspec.org/en).
+
+At a minimum, MAAP metadata should conform to STAC specification requirements. MAAP also requires the STAC-equivalent of certain [Unified Metadata Model (UMM)](https://www.earthdata.nasa.gov/about/esdis/eosdis/cmr/umm) metadata elements (UMM is a metadata schema used in [NASA's Common Metadata Repository (CMR)](https://www.earthdata.nasa.gov/about/esdis/eosdis/cmr)). These elements are included below, and are broken down at the collection and item level.
+
+**Note**: NASA CMR also uses Global Change Master Directory (GCMD) Keywords to help with consistency and searchability. As a best practice, MAAP recommends using GCMD keywords in applicable fields, which are noted below. For more information, see ["GCMD Keyword Viewer"](https://www.earthdata.nasa.gov/data/tools/idn/gcmd-keyword-viewer). Downloadable CSV files of GCMD keywords are also provided for relevant fields. 
+
+## STAC Collection-Level Metadata Fields
+- `id`: **Required**. The unique identifier of the collection.
+- `version`: **Required**. The version of the data. This is different than the `stac_version` element. 
+- `title`: **Required**. A short descriptive title of the datatset. This should differ from the `id`.
+- `sci:doi`: **Required if available**. The DOI of the collection should be provided using the [Scientific Citation Extension Specification](https://github.com/stac-extensions/scientific/tree/main). 
+  - For more information and resources about DOIs, see [Open Sourcing Code](../../science/oss_documentation/doi_and_licensing.ipynb).  
+- `description`: **Required** A detailed description or abstract for the dataset. The description should be useful to the science community, but also approachable for a first time user of the data. It is recommended to include information such as what platforms/instruments were/are used to collect the data, the parameters that are provided, the spatial and temporal coverage, and the purpose and/or intended use of the data.
+- `providers`: **Required**. The name of the organization(s) responsible for originating, processing, archiving, and/or distributing the dataset.
```

Comment:
can you provide some hints about which organizations should be listed for which roles? e.g. "if the data are hosted in a MAAP bucket, add the 'host' role to the "MAAP" provider"